doc/ansible/dev/apb_migration_guide.md

```diff
@@ -0,0 +1,241 @@
+* `<kind>` is the kind of your resource. For example, if you are creating a `memcached-operator`, your `kind` would likely be `Memcached`
+
+So for the example `memcached-operator`, the command would be:
+
+```
+operator-sdk new memcached-operator --type=ansible --api-version=apps.example.com/v1alpha1 --kind=Memcached
+```
+
+
+Once this is generated, take the `build`, `deploy`, and `molecule` directories, as well as the `watches.yaml` and copy them into your APB directory.
+
+#### Dockerfile
+You now have two Dockerfiles, your original APB `Dockerfile` at the top-level, and a `build/Dockerfile` for your operator. 
+
+In your `build/Dockerfile`, ensure that your playbooks and roles are being copied to `${HOME}/roles` and `${HOME}/playbooks`, and that your `watches.yaml` is being copied to `${HOME}/watches.yaml`. 
+
+If you are installing any additional dependencies, ensure that those are reflected in your `build/Dockerfile` as well. 
+
+As a sample, your `build/Dockerfile` will probably look something like this:
+
+```
+FROM quay.io/operator-framework/ansible-operator:v0.9.0
+
+COPY watches.yaml ${HOME}/watches.yaml
+
+COPY roles/ ${HOME}/roles/
+COPY playbooks/ ${HOME}/playbooks/
+```
+
+Once this is done you may remove your original APB `Dockerfile`.
+
+#### watches.yaml
+In the `watches.yaml`, ensure the playbook for your `kind` points to your `provision.yml` playbook in the container (likely location for that will be `/opt/ansible/playbooks/provision.yml`). 
+
+Next, add a finalizer block with a name of: `finalizer.<name>.<group>/<version>`, and set the playbook to point to your `deprovision.yml` in the container (likely location for that will be `/opt/ansible/playbooks/deprovision.yml`). For the memcached-operator we generated above, the watches.yaml would look like this:
+
+```yaml
+---
+- version: v1alpha1
+  group: apps.example.com
+  kind: Memcached
+  playbook: /opt/ansible/playbooks/provision.yml
+  finalizer:
+    name: finalizer.memcached.apps.example.com/v1alpha1
+    playbook: /opt/ansible/playbooks/deprovision.yml
+```
+
+##### Binding
+If you have a `bind` playbook, add a new entry to your `watches.yaml` (you can copy paste the existing entry). 
+
+The `version` and `group`, will remain unchanged, but update the `kind` with a `Binding` suffix. 
+
+For example, if you have a resource with `kind: Memcached`, the kind of your new entry will be `MemcachedBinding`. 
+
+The playbook for this entry should map to your `bind` playbook, (likely location `/opt/ansible/playbooks/bind.yml`), and if you have an `unbind` playbook then set the playbook for the finalizer to point to it (likely location `/opt/ansible/playbooks/unbind.yml`). If you don't have an `unbind` playbook, remove the finalizer block for your `Binding` resource.
```

Comment:
How important is it that the finalizer for `Binding` be removed if there is no unbind? If it is super important, I'd find a way to emphasize that better. Bold the text or something.